appinventor/components/src/com/google/appinventor/components/runtime/util/ErrorMessages.java

```diff
@@ -621,6 +624,10 @@
     // REPL errors
     errorMessages.put(ERROR_UNABLE_TO_INSTALL_PACKAGE,
         "Unable to launch the package installer for %1$s.");
+
+    // Augmented Reality errors
+    errorMessages.put(ERROR_INVALID_CONFIGURATION_VALUE,
+            "Invalid value %1$d given for ARConfigurationType.  Valid settings are 1, 2, or 3.");
```

Comment:
@nicholec It would be good for the indentation on this line to match those of previous entries.

Reply:
Updated the indentation level